docs/reference-guides/interactivity-api/README.md

```diff
@@ -19,7 +19,7 @@ Use the following links to locate the topic you're interested in. If you have ne
 - **[Requirements](#requirements-of-the-interactivity-api):** Check this section before you start creating your interactive blocks with the Interactivity API.
 - **[Quick Start Guide](https://developer.wordpress.org/block-editor/reference-guides/interactivity-api/iapi-quick-start-guide/):** Get a custom block using the Interactivity API up and running in less than one minute.
 - **[Tutorial: A first look at the Interactivity API](https://developer.wordpress.org/news/2024/04/11/a-first-look-at-the-interactivity-api/)** This article from the [WordPress Developer Blog](https://developer.wordpress.org/news/) is a great way to get introduced to the Interactivity API.
-- **[API Reference](https://developer.wordpress.org/block-editor/reference-guides/interactivity-api/api-reference/):** To take a deep dive into how the API works internally, the list of Directives, and how the Store works.
+- **[Directives and Store API Reference.](https://developer.wordpress.org/block-editor/reference-guides/interactivity-api/directives-store-api-reference/):** To take a deep dive into how the API works internally, the list of Directives, and how the Store works.
```

Comment:
Hey @piyush-sri ,

The embedded link does not exist. The correct URL is: https://developer.wordpress.org/block-editor/reference-guides/interactivity-api/api-reference/